spec/brainstem-model-spec.coffee

```diff
@@ -4,6 +4,109 @@ describe 'Brainstem.Model', ->
   beforeEach ->
     base.data.reset()
 
+  describe 'instantiation', ->
+    newModel = null
+
+    itReturnsCachedInstance = ->
+      it 'returns cached instance', ->
+        expect(newModel).toEqual model
```

Comment:
Just curious: would comparing `cid`s be a bit more assertive or does equality checking the whole objects accomplish the same things?